Composer/packages/client/src/components/Header.tsx

```diff
@@ -95,43 +139,121 @@ export const Header = () => {
   const locale = useRecoilValue(localeState(projectId));
   const appUpdate = useRecoilValue(appUpdateState);
   const { showing, status } = appUpdate;
+  const [showStartBotsPanel, setStartPanelView] = useState(false);
+  const runningBots = useRecoilValue(runningBotsSelector);
+  const [startPanelText, setStartPanelText] = useState('');
+  const { projectIds: projectIdsRunning } = runningBots;
+  const [areBotsStarted, setBotsInBotProjectStarted] = useState<boolean>(false);
+  const { stopAllBots, startAllBots } = useLocalBotOperations();
+  const [showStartBotsWidget, setStartBotsWidgetVisible] = useState(true);
+  const currentMode = useRecoilValue(currentModeState);
+
+  useEffect(() => {
+    // TODO: Start using modes to detect current page after #4361
+    if (currentMode !== 'home') {
+      setStartBotsWidgetVisible(true);
+      return;
+    }
+    setStartBotsWidgetVisible(false);
+  }, [currentMode]);
+
+  useEffect(() => {
+    if (projectIdsRunning.length > 0) {
+      setBotsInBotProjectStarted(true);
+    }
+  }, [projectIdsRunning]);
+
+  useEffect(() => {
+    if (runningBots.projectIds.length > 0) {
+      setStartPanelText(
+        `${formatMessage('Stop all bots')} (${runningBots.projectIds.length}/${runningBots.totalBots} ${formatMessage(
```

Comment:
@beyackle  Can i do this for dynamic text?

Reply:
```js
return formatMessage('Stop all bots ({running}/{total}) running', {
        running: runningBots.projectIds.length,
        total: runningBots.totalBots,
      });
```